.gitlab-ci.yml

```diff
@@ -7,6 +7,8 @@ stages:
 
 variables:
   # some default values
+  CACHEKEY: bionic-V2018-04-28-B
```

Comment:
I don't like having some crazy ad-hoc thing for the key.
Just use a timestamp or an incrementing number.

Reply:
That's a timestamp.

Reply:
format = `$IMAGE-v$DATE-$VERSION`

Reply:
Where `$REVISION` is needed as to avoid shooting myself in the foot today.

Reply:
Don't put image in it, and if you put the date in it just make it precise enough to serve as the whole thing.

Reply:
> Don't put image in it,

Why? It seems useful to me.